apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -659,11 +661,19 @@ const BookingPage = ({
                       ? "-mt-4"
                       : ""
                   )}>
+                <Turnstile
+                    className="fixed bottom-4 right-4"
+                    onError={() => setStatus("error")}
+                    onExpire={() => setStatus("expired")}
+                    onSuccess={() => setStatus("solved")}
+                    siteKey="0x4AAAAAAAFM2dMLqlOd35hp"
```

Comment:
You've exposed the site key here, this should be in a .env file

Reply:
ideally this should actually be added in the App Config (see for example how the GTM app works or the Fathom App). You would make an event-type App and then add your key in the input box

Reply:
@hariombalhara we should maybe write an abstraction/adapter to enable all kinds of captchas  the same way we offer all kinds of analytics apps